features/step_definitions/feedback_steps.rb

```diff
@@ -2,8 +2,8 @@
   expect(page).to have_link('Feedback')
 end
 
-Then(/^I should see a link to get in touch$/) do
-  expect(page).to have_content('Get In Touch')
+Then(/^I should see an email link to get in touch$/) do
+  expect(page).to have_content('micropurchase@gsa.gov')
```

Comment:
@jessieay that stackoverflow implementation seems like overkill for this instance, but all for using the `has_xlink` method. more specific.